src/GeneralStateTestsFiller/stEWASMTests/validateBytecodeMissingVersionTwoBytesFromTxIsInvalidFiller.yml

```diff
@@ -0,0 +1,43 @@
+# Test that init bytecode (with magic + two byte version) from tx is invalid
+validateBytecodeMissingVersionTwoBytesFromTxIsInvalid:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    # tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          balance: '99916113919'
+        # newly-created contract
+        # expect code to be empty since create should fail
+        6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+          shouldnotexist: 1
+  transaction:
+    data: [
+      # Wasm magic (0061736d) with only one byte for version number
+      "0x0061736d0100"
```

Comment:
This seems to be 2 bytes for version?

Reply:
Ah ok, the file is named 2 bytes, but the comment is wrong.

Reply:
good catch, fixed